walkthrough/garden_cookiecutter/{{cookiecutter.namespace}}/{{cookiecutter.version}}/{{cookiecutter.short_name}}.py

```diff
@@ -14,20 +15,14 @@
 
 def load_countries_regions() -> Table:
```

Comment:
I think these auxiliary loading functions should not exist. The lines `paths.load_dataset(...)[...]` should be added as part of `run`, right after loading the main table.

Reply:
I checked, and no one is using these options anyway. I removed them altogether.